news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,175 @@
+
+Traditionally, osu! has not had accuracy on slider heads, and so accuracy pp have excluded sliders in calculations. However, the lazer client has slider head accuracy, and so accuracy pp now includes sliders on scores played with slider head accuracy.
+
+This change may not affect all scores set with slider head accuracy, as accuracy pp also has an object count cap, meaning accuracy pp will go unchanged if the cap was already reached. Additionally, very low-accuracy scores may not be boosted enough by the added object counts to result in any extra reward.
+
+We expect more pp changes related to slider head accuracy in the future!
+
+### Minor changes
+
+- A [slider mechanics change](https://github.com/ppy/osu/pull/24966) was made to adjust slider ends to be more lenient during fast sliders, which has resulted in buffs on some maps with very fast sliders such as [ocelot - KAEDE](https://osu.ppy.sh/beatmapsets/660630#osu/1398809). You can find more about this change in the related [osu!(lazer) updates video](https://www.youtube.com/watch?v=SlWKKA-ltZY).
+- [Givikap120](https://osu.ppy.sh/users/10560705) refactored and cleaned up [various](https://github.com/ppy/osu/pull/29293) [areas](https://github.com/ppy/osu/pull/29294) [of](https://github.com/ppy/osu/pull/29292) [performance calculation code](https://github.com/ppy/osu/pull/29291) to make them easier to understand and maintain.
+- An [aim skill multiplier rebalance](https://github.com/ppy/osu/pull/29998) proposed by [tsunyoku](https://osu.ppy.sh/users/11315329) was made in order to ensure the final results of all major changes are in line with expectations.
+- A [speed accuracy scaling change](https://github.com/ppy/osu/pull/30088) proposed by [StanR](https://osu.ppy.sh/users/7217455) was made to punish lower accuracies below OD 8 more harshly.
+- [Finadoggie](https://osu.ppy.sh/users/14182048)'s proposed [score statistics estimation change](https://github.com/ppy/osu/pull/27691) was made to ensure that slider breaks and dropped slider ends are correctly accounted for in osu!(lazer) scores.
+
+## osu!taiko
+
+In order to aid understanding of the changes to osu!taiko, these are the skills in difficulty calculation which will be referenced throughout:
+
+- **Stamina**: the speed at which you hit notes, based on an assumed finger count of 2 per colour
+- **Colour**: the frequency of which the beatmap changes between a don or kat
+- **Rhythm**: the complexity of the beatmap's rhythm in relation to notes' independent rhythm ratios
+
+### Consideration for TL tapping in the stamina component of difficulty calculation
+
+With the recent update to osu!taiko's difficulty calculation, an issue has arisen concerning a unique playstyle known as TL-tapping. [Maxie's video](https://www.youtube.com/watch?v=WM7gkFBDl9s) provides a more in-depth explanation of this playstyle.
```

Comment:
>  With the recent update to osu!taiko's difficulty calculation

what is this referring to?

Reply:
Pretty sure this is referring to https://osu.ppy.sh/home/news/2022-09-28-changes-to-osu-taiko-sr-and-pp. I think I just overlooked that part when taking it from taiko committee